fuse_models/src/unicycle_2d.cpp

```diff
@@ -204,6 +204,10 @@ void Unicycle2D::onInit()
   process_noise_covariance_ = fuse_core::Vector8d(process_noise_diagonal.data()).asDiagonal();
 
   private_node_handle_.param("scale_process_noise", scale_process_noise_, scale_process_noise_);
+  private_node_handle_.param("rotate_process_noise_covariance_to_state_orientation",
+                            rotate_process_noise_covariance_to_state_orientation_,
+                            rotate_process_noise_covariance_to_state_orientation_);
```

Comment:
Indentation seems to be missing one space for each line here:
```suggestion
                             rotate_process_noise_covariance_to_state_orientation_,
                             rotate_process_noise_covariance_to_state_orientation_);
```